From a7ebecb9fbee2190e649c44ed53f1299013baa30 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Sun, 18 Mar 2018 11:56:57 +0100
Subject: [PATCH] Fix build with Qt 5.11 (missing headers)

Reviewers: #calligra:_3.0, anthonyfieroni, danders

Reviewed By: #calligra:_3.0, anthonyfieroni, danders

Subscribers: anthonyfieroni, danders

Tags: #calligra:_3.0

Differential Revision: https://phabricator.kde.org/D11454

--- libs/widgets/KoCsvImportDialog.cpp.orig	2018-01-26 09:24:44 UTC
+++ libs/widgets/KoCsvImportDialog.cpp
@@ -21,6 +21,7 @@
 #include "KoCsvImportDialog.h"
 
 // Qt
+#include <QButtonGroup>
 #include <QTextCodec>
 #include <QTextStream>
 
--- libs/widgets/KoPageLayoutWidget.cpp.orig	2018-01-26 09:24:44 UTC
+++ libs/widgets/KoPageLayoutWidget.cpp
@@ -23,6 +23,8 @@
 
 #include <KoUnit.h>
 
+#include <QButtonGroup>
+
 class Q_DECL_HIDDEN KoPageLayoutWidget::Private
 {
 public:
--- plugins/chartshape/dialogs/TableEditorDialog.cpp.orig	2018-01-26 09:24:44 UTC
+++ plugins/chartshape/dialogs/TableEditorDialog.cpp
@@ -24,6 +24,7 @@
 
 // Qt
 #include <QAbstractItemModel>
+#include <QAction>
 
 // Calligra
 #include <KoIcon.h>
--- plugins/formulashape/FormulaToolWidget.cpp.orig	2018-01-26 09:24:44 UTC
+++ plugins/formulashape/FormulaToolWidget.cpp
@@ -30,6 +30,7 @@
 #include <QWidgetAction>
 #include <QTableWidget>
 #include <QAction>
+#include <QHeaderView>
 #include <QMenu>
 
 FormulaToolWidget::FormulaToolWidget( KoFormulaTool* tool, QWidget* parent )
--- sheets/dialogs/LayoutDialog.cpp.orig	2018-01-26 09:24:44 UTC
+++ sheets/dialogs/LayoutDialog.cpp
@@ -36,6 +36,7 @@
 #include <math.h>
 
 #include <QIntValidator>
+#include <QButtonGroup>
 #include <QCheckBox>
 #include <QFrame>
 #include <QLabel>
--- words/part/dialogs/KWAnchoringProperties.cpp.orig	2018-01-26 09:24:44 UTC
+++ words/part/dialogs/KWAnchoringProperties.cpp
@@ -35,6 +35,7 @@
 
 #include <kundo2command.h>
 
+#include <QButtonGroup>
 #include <QComboBox>
 
 const int KWAnchoringProperties::vertRels[4][20] = {
--- words/part/dialogs/KWRunAroundProperties.cpp.orig	2018-01-26 09:24:44 UTC
+++ words/part/dialogs/KWRunAroundProperties.cpp
@@ -28,6 +28,8 @@
 
 #include <kundo2command.h>
 
+#include <QButtonGroup>
+
 KWRunAroundProperties::KWRunAroundProperties(FrameConfigSharedState *state)
         : m_state(state)
 {
